tests/translate/translate_d_sw.py

```diff
@@ -68,23 +68,24 @@ def compute(self, inputs):
         return self.slice_output(inputs)
 
 
-class TranslateUbKE(TranslateFortranData2Py):
-    @gtstencil()
-    def ubke(
-        uc: FloatField,
-        vc: FloatField,
-        cosa: FloatFieldIJ,
-        rsina: FloatFieldIJ,
-        ut: FloatField,
-        ub: FloatField,
-        dt4: float,
-        dt5: float,
-    ):
-        with computation(PARALLEL), interval(...):
-            ub = d_sw.ubke(uc, vc, cosa, rsina, ut, ub, dt4, dt5)
+@gtstencil
+def ubke(
+    uc: FloatField,
+    vc: FloatField,
+    cosa: FloatFieldIJ,
+    rsina: FloatFieldIJ,
+    ut: FloatField,
+    ub: FloatField,
+    dt4: float,
+    dt5: float,
+):
```

Comment:
Good find! Can you also switch these from using a decorator to the functional version inside the `__init__`? Then these won't be generated as pytest scans through the file.

Reply:
Done when merging with master.